MANAS/Javascript/Assignment 2/ques2.html

```diff
@@ -0,0 +1,82 @@
+                <div class="col-sm-8">
+                    <input type="number" name="sub3" id="sub3" required>
+                </div>
+            </div>
+            <div class="row m-3">
+                <label for="sub4" class="col-sm-3"><b>Enter Science marks:</b></label>
+                <div class="col-sm-8">
+                    <input type="number" name="sub4" id="sub4" required>
+                </div>
+            </div>
+            <div class="row m-3">
+                <label for="sub5" class="col-sm-3"><b>Enter Social Science marks:</b></label>
+                <div class="col-sm-8">
+                    <input type="number" name="sub5" id="sub5" required>
+                </div>
+            </div>
+            <div class="m-5">
+                <button id="grade" onclick="percent()" class="btn btn-success p-1 ml-lg-2" style="width: 150px;" type="button">Click Me !!!</button>
+            </div>            
+            <h1 id="pid"></h1>
+            <h1 id="pid1"></h1>
+        </div>
+        <script>
+            function percent(){
+                var a, b, c, d, e, total, percentage ;
+                a = Number(document.getElementById("sub1").value) ;
+                b = Number(document.getElementById("sub2").value) ;
+                c = Number(document.getElementById("sub3").value) ;
+                d = Number(document.getElementById("sub4").value) ;
+                e = Number(document.getElementById("sub5").value) ;               
+                total = a + b + c + d + e;
+                document.getElementById("pid").innerText = " Total Marks secured is: " + total ;
+                percentage = (total/500) * 100 ;
+                console.log(percentage)
+                if(percentage <= 100 && percentage >=75){
+                    document.getElementById("pid1").innerHTML = "Grade is: A" ;
+                }
+                else if(percentage <= 74 && percentage >=60){
+                    document.getElementById("pid1").innerHTML = "Grade is: B" ;
+                }
+                else if(percentage <= 59 && percentage >=40){
+                    document.getElementById("pid1").innerHTML = "Grade is: C" ;
+                }
+                else if(percentage <= 40 && percentage >=30){
+                    document.getElementById("pid1").innerHTML = "Grade is: D" ;
+                }
+                else{
+                    document.getElementById("pid1").innerHTML = "Grade is: F" ;
+                }
+            document.getElementById("pid1").innerHTML += "<br>Your Percentage is :" + percentage ;
+        }
+        </script>        
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    </body>
```

Comment:
grade is not calculated correctly..also the code is exactly copied

Reply:
yes mam plz check my branch i have already changes my file and also my grade is calculated correctly